controllers/spec/common.go

```diff
@@ -1072,7 +1072,7 @@ func getProcessJavaRuntimeArgs(name, packageName, clusterName, logLevel, details
 	state *v1alpha1.Stateful,
 	tlsConfig TLSConfig, authConfig *v1alpha1.AuthConfig,
 	maxPendingAsyncRequests *int32, logConfigFileName string) []string {
-	classPath := "/pulsar/instances/java-instance.jar"
+	classPath := "/pulsar/instances/java-instance.jar:/pulsar/lib/*"
```

Comment:
log4j yaml extensions requires some additional lib such as `com.fasterxml.jackson.core-jackson-databind-2.13.2.1.jar` and `com.fasterxml.jackson.dataformat-jackson-dataformat-yaml-2.13.2.jar`